extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLConfig.java

```diff
@@ -59,6 +60,35 @@
     @ConfigItem(defaultValue = "Default")
     TypeAutoNameStrategy autoNameStrategy;
 
+    /**
+     * List of extension fields that should be included in the error response.
+     * By default none will be included. Examples of valid values include
+     * [exception,classification,code,description,validationErrorType,queryPath]
+     */
+    @ConfigItem
+    Optional<List<String>> errorExtensionFields;
+
+    /**
+     * List of Runtime Exceptions class names that should show the error message.
+     * By default Runtime Exception messages will be hidden and a generic `Server Error` message will be returned.
+     */
+    @ConfigItem
+    Optional<List<String>> showRuntimeExceptionMessage;
+
+    /**
+     * List of Checked Exceptions class names that should hide the error message.
+     * By default Checked Exception messages will show the exception message.
+     */
+    @ConfigItem
+    Optional<List<String>> hideCheckedExceptionMessage;
```

Comment:
If the MP counterpart is `mp.graphql.hideErrorMessage` then maybe this should be named `hideErrorMessage` too (rather than `hideCheckedExceptionMessage`)? To be less confusing?
Dtto for `showRuntimeExceptionMessage`

Reply:
I actually named it different because I think the MP one is not named well....